### PR TITLE
tokenbalancehistory

### PR DESCRIPTION
**Pull request by bumbummen99**
Added `tokenbalancehistory` action method to token.
This is a Pro endpoint and does require Pro API access.

### CODE REVIEW

File: src/token.ts [hunk 1]
```diff
@@ -39,8 +39,28 @@ function getAccountBalanceForTokenContractAddress(
   )
 }
 
+function getHistoricAccountBalanceForTokenContractAddress(
```
Comment (jpgarcia) [inline src/token.ts]: I'd rename this as `getHistoricialAccountBalanceForTokenContractAddress` to be consistent with the docs

**Review by jpgarcia — CHANGES_REQUESTED** [review]

File: src/token.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ function getAccountBalanceForTokenContractAddress(
   )
 }
 
-function getHistoricAccountBalanceForTokenContractAddress(
+function getHistoricialAccountBalanceForTokenContractAddress (
```
Comment (jpgarcia) [inline src/token.ts]: It seems you missed the rename in the `export default` section

**Review by jpgarcia — CHANGES_REQUESTED** [review]

File: src/token.ts (hunk 2 above)
Comment (jpgarcia) [reply src/token.ts]: While reviewing it I also noticed that I misspelled the `Historicial` word!! it should be `getHistoricalAccountBalanceForTokenContractAddress`

Sorry about that!

Comment (bumbummen99) [reply src/token.ts]: No problem :) Fixing it rn

**Review by jpgarcia — APPROVED** [review]

**Pull request merged**